spec.html

```diff
@@ -4912,9 +4912,11 @@ <h1>IteratorClose ( _iteratorRecord_, _completion_ )</h1>
         1. Assert: Type(_iteratorRecord_.[[Iterator]]) is Object.
         1. Assert: _completion_ is a Completion Record.
         1. Let _iterator_ be _iteratorRecord_.[[Iterator]].
-        1. Let _return_ be ? GetMethod(_iterator_, `"return"`).
-        1. If _return_ is *undefined*, return Completion(_completion_).
-        1. Let _innerResult_ be Call(_return_, _iterator_, &laquo; &raquo;).
+        1. Let _innerResult_ be GetMethod(_iterator_, `"return"`).
+        1. If _innerResult_.[[Type]] is ~normal~,
+          1. Let _return_ be _innerResult_.[[Value]].
+          1. If _return_ is *undefined*, return Completion(_completion_).
+          1. Set _innerResult_ to Call(_return_, _iterator_, &laquo; &raquo;).
```

Comment:
Last argument of [`Call`](https://tc39.es/ecma262/#sec-call) defaults to empty list.
Please also see https://github.com/tc39/ecma262/pull/1414.